systers_portal/systers_portal/settings/testing.py

```diff
@@ -11,11 +11,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': config('DB_NAME', default='systersdb'),
-        'USER': config('DB_USER'),
-        'PASSWORD': config('DB_PASSWORD'),
-        'HOST': config('DB_HOST', default='localhost'),
-        'PORT': config('DB_PORT', default=5432, cast=int),
+        'NAME': config('TEST_DB_NAME', default='systersdb'),
```

Comment:
Question: why not keep the same names?

So basically the same names for beth dev and testing, is that causing issues?

Reply:
Correct me if I am wrong. Is testing database is same as the developer using main database, because testing database is create and destroyed while running testcases.